UPGRADE-8.0.md

```diff
@@ -152,6 +152,45 @@ FrameworkBundle
    $application->addCommand(new CreateUserCommand());
    ```
 
+HttpFoundation
+--------------
+
+ * Remove the following deprecated session options from `NativeSessionStorage`: `referer_check`, `use_only_cookies`, `use_trans_sid`, `sid_length`, `sid_bits_per_character`, `trans_sid_hosts`, `trans_sid_tags`
+
+   *Before*
+   ```php
+   use Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage;
+
+   $storage = new NativeSessionStorage([
+       'referer_check' => 'http://example.com',
+       'use_only_cookies' => true,
+       'use_trans_sid' => false,
+       'sid_length' => 48,
+       'sid_bits_per_character' => 6,
+       'trans_sid_hosts' => 'example.com',
+       'trans_sid_tags' => 'a=href,area=href',
+   ]);
+   ```
+
+   *After*
+   ```php
+   use Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage;
+
+   // These options are no longer supported and should be removed
+   $storage = new NativeSessionStorage([
+       // Configure other session options as needed
+   ]);
+   
+   // Use PHP's session configuration functions directly if needed:
```

Comment:
not sure that's appropriate advice: these have also been deprecated from php

Reply:
makes sense